src/libs/Localize/index.js

```diff
@@ -2,12 +2,26 @@ import _ from 'underscore';
 import lodashGet from 'lodash/get';
 import Str from 'expensify-common/lib/str';
 import * as RNLocalize from 'react-native-localize';
+import Onyx from 'react-native-onyx';
 import Log from '../Log';
 import Config from '../../CONFIG';
 import translations from '../../languages/translations';
 import CONST from '../../CONST';
 import LocaleListener from './LocaleListener';
 import BaseLocaleListener from './LocaleListener/BaseLocaleListener';
+import ONYXKEYS from '../../ONYXKEYS';
+
+let userEmail = '';
+Onyx.connect({
+    key: ONYXKEYS.SESSION,
+    waitForCollectionCallback: true,
+    callback: (val) => {
+        if (!val) {
+            return;
+        }
+        userEmail = val.email;
+    },
+});
```

Comment:
I think we can create some comment to address why are we need user email here - similar to `LocaleListener.connect()` comment below